src/js/module.js

```diff
@@ -204,18 +202,36 @@ iotjs_module_t.load = function(id, parent) {
   return module.exports;
 };
 
+iotjs_module_t.loadRemote = function(id, parent) {
```

Comment:
Two questions:
* Do we need the parent argument at all?
* Can we have parameters like this: "function(filePath, source)"?
  * Maybe we'll need to rename this then to something else.

Reply:
Good point, parent would be `null` anyways.

Reply:
I don't think `filePath` would be a proper name, `fileName` would fit this usecase more, since it never really exists, so doesn't have a path.